swayidle/swayidle.1.txt

```diff
@@ -0,0 +1,59 @@
+/////
+vim:set ts=4 sw=4 tw=82 noet:
+/////
+:quotes.~:
+
+swayidle (1)
+============
+
+Name
+----
+swayidle - Screen power manager for sway
+
+Synopsis
+--------
+'swayidle' [options] [timeout <seconds> exec <parameter>
+			[resume exec <parameter>]] ...
+
+Swayidle handles idle events from sway, and setup lockscreen and turns off
```

Comment:
"from sway: it sets up the lockscreen and turns off the screen." might be better, but this isn't as clear cut